### PR TITLE
Adding webpage and initialize settings

### LINKED ISSUE #15: Add project landing page and handle view in Django
@ohadrevach
Project landing page and corresponding django code to handle it need to be added.

### PR DESCRIPTION
**Pull request by ohadrevach**
Creating web page application using the django,
set up it's requirements and add index.html
Closes #15

### CODE REVIEW

**Review by GilShechter — DISMISSED** [review]
Looks good to me

**Review by efraimvis — APPROVED** [review]
Looks good.

**Review by efraimvis — CHANGES_REQUESTED** [review]
Please link this PR to issue #15

**Review by efraimvis — DISMISSED** [review]

**Review by GalOved — COMMENTED** [review]
In index.html i add a review.

**Review by WaseemAbuSobe — APPROVED** [review]

**Comment by WaseemAbuSobe** [comment]
looking good

**Review by WaseemAbuSobe — APPROVED** [review]
looking good

**Comment by liornoy** [comment]
@ohadrevach did you see Gal's review?

**Review by natifridman — COMMENTED** [review]
I guess `webpage` is a temporary name?

**Comment by GilShechter** [comment]
> I guess `webpage` is a temporary name?

Well, we named the project "tasties" so we have a directory with this name.
Then we created the application for the web page, so, what should we call it? 
The project name is already taken.

**Comment by natifridman** [comment]
> > I guess `webpage` is a temporary name?
> 
> Well, we named the project "tasties" so we have a directory with this name. Then we created the application for the web page, so, what should we call it? The project name is already taken.

Let's keep it with this name for now, but think about it when you will work on the models

File: webpage/templates/webpage/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		Tasties is a platform for sharing and getting the best recieps that everyone can make.<br>
+		You can get and see feedbacks from Tasties community and by that improve your cooking skills and recieps.<br><br>
+		Dont know what to choose?<br>
+		In our platform you can search receips by your ingredients you like and rate the recipe so others can enjoy too.
+	</p>
+	</h2>
+
+
+</body>
+</html>
```
[the remark this answers is not in the document]
Comment (ohadrevach) [reply webpage/templates/webpage/index.html]: Fixed

**Review by natifridman — DISMISSED** [review]
Please squash the commits so we can merge it

**Review by WaseemAbuSobe — DISMISSED** [review]

**Review by efraimvis — APPROVED** [review]
I can see you squashed the commits together, I don't see any probelms.

**Review by GalOved — APPROVED** [review]
All looking good to me.

**Review by svatares — APPROVED** [review]

**Pull request merged**